unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioStream.cs

```diff
@@ -64,7 +64,7 @@ private bool IsPlayerInSameSceneAsComponent(string currentSceneId)
         {
             if (scene == null) return false;
             if (string.IsNullOrEmpty(currentSceneId)) return false;
-            return scene.sceneData.id == currentSceneId;
+            return (scene.sceneData.id == currentSceneId) || (scene is GlobalScene && ((GlobalScene)scene).isPortableExperience);
```

Comment:
I think we can optimize a little using `is` and removing the casting, also it might be little bit more clear. wdyt?
```suggestion
            return (scene.sceneData.id == currentSceneId) || (scene is GlobalScene globalScene && globalScene.isPortableExperience);
```

Reply:
Thank you Pato, I didn't know that we can do this! `scene is GlobalScene globalScene`. Done!